src/config/seo.config.ts

```diff
@@ -0,0 +1,103 @@
+import { Metadata } from 'next'
+
+type Page = 'landing' | 'cashout' | 'send' | 'requestCreate' | 'terms' | 'privacy' | 'jobs'
```

Comment:
thought: hmmm, this duplicates the page definition. Now we have these page definitions, but they're also defined by the folder layout in next js. What if we change the name of send to something else? Also, relevant, now we merged terms and privacy as you can see in this PR: https://github.com/peanutprotocol/peanut-ui/pull/584/files . However, this page didn't get updated.

Suggestion: can we think of a different way to define the pages? Frankly, I think the setup of having the metadata defined close to the page (e.g. in src/app/cashout/page.tsx or /metadata.tsx) is a better abstraction